### ISSUE TITLE
Send watermeter stats daily

### ISSUE THREAD

**Issue by bartrylant** [Th3S4mur41/hw2energyid#14]
Is there an option to import the watermeter stats daily? (On MacOS or with a Synology server?)

**Comment by Th3S4mur41** [comment]
Hi @bartrylant,
not yet, but I'd be happy to add it 😉
Could you copy an output of the water meter api in here to help me figure out the data structure?

**Comment by bartrylant** [comment]
Sure
```
{"wifi_ssid":"xxxxx","wifi_strength":98,"total_liter_m3":0.038,"active_liter_lpm":0,"total_liter_offset_m3":0}
```
The total_liter_m3 is the total, since the meter started. It would also be nice if it would be able to set a "starter" setting for the meter, so I have the correct numbers in energieID. E.g. my meter was at 00331,725 when I installed the homewizard meter. I would like to have this reflected in energieID

**Comment by Th3S4mur41** [comment]
Thanks for the clarification

I suppose what you are asking for will be covered by `total_liter_offset_m3` in the future, but as [their docs mention](https://homewizard-energy-api.readthedocs.io/endpoints.html#data-points-for-hwe-wtr), this is currently under development and should not be used yet.

I'll give it a try

**Comment by Th3S4mur41** [comment]
@bartrylant Could you try to run the following cmd?
This is just a dry run to check the reading...

`npx --yes hw2energyid@1.3.0-alpha.1 --meter <watermeter id> --energyid <webhook url> -o 331.725 --dry-run`

If you want to try and send a first record to energyid, just run the same script without the last argument

**Comment by Th3S4mur41** [comment]
Hey @bartrylant thanks for the feedback.
The feature is now released in version [1.3.2](https://github.com/Th3S4mur41/hw2energyid/releases/tag/v1.3.2)

If you want a to make sure the script does the update every day, you can run it in recurrent mode with the `-r` option:

```sh
npx --yes hw2energyid@latest --meter <watermeter id> --energyid <webhook url> -o 331.725 -r
```

Or if you prefer to run it on your Synology, you can also use the docker image as [documented in the README](https://github.com/Th3S4mur41/hw2energyid#docker)

See the following link for a guide to run docker images on Synaology NAS:  
https://www.wundertech.net/how-to-use-docker-on-a-synology-nas/
